app/services/ZuoraCatalogService.scala

```diff
@@ -0,0 +1,41 @@
+package services
+
+import services.aws.AwsS3Client.{fetchJson, s3}
+import com.amazonaws.services.s3.model.GetObjectRequest
+import com.gu.support.config.{Stage, Stages}
+import io.circe.generic.auto._
+import io.circe.syntax._
+import models.ZuoraCatalog._
+
+object ZuoraCatalogService {
+
+  val paperCollectionProductRatePlanIds = List(
```

Comment:
Why did we decide to hardcode the rate plan IDs and not use the Product Benefit tag on the Product Rate Plan Charge to map the products like what happens in Membership-Common?

Reply:
because I didn't knew about it! will have a look